External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -60,20 +70,45 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 	end
 end
 
+% JSON saving

```

Comment:
That's a copy of what we have above. We can delete this. But we can of course introduce the bSaveJson variable.

Reply:
Fixed in